1-Draft/RFCNNNN-Multi-Line-Continuance-For-Commands.md

```diff
@@ -110,6 +110,40 @@ For example, consider this syntax:
 
 That command will not parse because there is no closing brace for the script block. What appears to be a closing brace is placed after the stop-parsing sigil, and therefore treated as an argument to the plink command. To correct this, the closing brace must be placed on a separate line, but in a multi-line command you cannot do that (because the command is multi-line, so where would the parser terminate after a stop-parsing sigil) and therefore, unless the sigil used to identify the end of the multi-line parameter/arguments was one that could be respected by the stop-parsing sigil, and safely be introduced without risk to breaking changes, enclosures simply cannot be used.
 
+### Allowing multi-line continuance for more than just commands
+
+Aside from easier multi-line wrapping parameters/arguments, there are scenarios where it would be useful to have easier multi-line wrapping for other statements as well, and some members of the community have [requested support for these scenarios](https://github.com/PowerShell/PowerShell/issues/9813). PowerShell currently supports line wrapping for objects when the line ends with the dot-reference operator, like this:
+
+```PowerShell
+$string.
+    ToUpper().
+    Trim().
+    Length
+```
+
+Other languages like C#, where an explicit line terminator is used, allow users to wrap such lines with the dot-reference operator at the beginning of a line, which looks something like this:
+
+```PowerShell
+$string
+    .ToUpper()
+    .Trim()
+    .Length
+```
+
+While the Parser could handle looking ahead for methods with arguments (round brackets), there is a problem with members or with methods with no arguments (which can be invoked in PowerShell to get the method signature(s)), because you can create a function with a name that starts with a dot (e.g. `function .dot {'see?'}; .dot`).
```

Comment:
But what about for piping output? A common pattern - with backticks - is something like:

```powershell
dir . -filter *.txt -recurse | `
  ? Length -gt 500kb | `
  copy -dest 'c:\bar'
```
Given the same general pattern you mention, at least with lines terminating with a pipe character you should be able to continue lines automatically (i.e. if you're piping to nothing explicit).

Reply:
This has always worked (or for a very long time, at least). The backticks aren't even necessary.

Reply:
Plus starting with PowerShell 6.2 you can even place your pipe symbols at the start of a line. Before PowerShell 6.2 that would require backticks, but now it just works without them.

Reply:
Sorry, but I gotta say it...

Your comment regarding PS 7 matters to far less than 1% of PowerShell users.

And will continue to matter to far less than 1% for years to come.

Reply:
All well and good, but progress is not made by dwelling on the past. 🙂